redash/models/base.py

```diff
@@ -26,8 +28,12 @@ def apply_pool_defaults(self, app, options):
             # Remove options NullPool does not support:
             options.pop("max_overflow", None)
 
+md = None
+if os.environ.get('REDASH_NAMESPACE'):
```

Comment:
Minor nit, this would follow convention if it was defined in settings and maybe as `SQLALCHEMY_DATABASE_SCHEMA`

Reply:
done